styles/tokens/footer.json

```diff
@@ -66,14 +66,34 @@
         "file": "settings",
         "themable": true
       },
-      "hover-color": {
-        "comment": "Toggle trigger hover color on the Footer.",
-        "value": "{purple-light.value}",
-        "attributes": {
-          "category": "color"
+      "hover": {
+        "color": {
+          "comment": "Toggle trigger hover color on the Footer.",
+          "value": "{white.value}",
+          "attributes": {
+            "category": "color"
+          },
+          "file": "settings",
+          "themable": true
         },
-        "file": "settings",
-        "themable": true
+        "font-weight": {
+          "comment": "Toggle trigger hover font-weight on the Footer.",
+          "value": "500",
+          "attributes": {
+            "category": "size"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "text-decoration": {
+          "comment": "Toggle trigger hover text decoration on the Footer.",
```

Comment:
verify that we mentioned the footer link color changes in the release notes @bobwise

Reply:
Checked. It's there.